api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -66,13 +66,13 @@ message UdpProxyConfig {
 
     // The match tree to use when resolving route actions for incoming requests.
     // See :ref:`Routing <config_udp_listener_filters_udp_proxy_routing>` for more information.
-    xds.type.matcher.v3.Matcher matcher = 9
+    xds.type.matcher.v3.Matcher matcher = 3
```

Comment:
please revert all the re-numbering and only add new field with `10`, otherwise it break backwards compatibilty.

Reply:
OK, fixed, thanks!